client/state/sites/domains/test/fixture.js

```diff
@@ -17,7 +17,7 @@ export const SITE_ID_SECOND = 77203074;
 
 // testing primary-domain
 export const DOMAIN_PRIMARY = {
-	autoRenewalDate: 'February 7, 2017',
```

Comment:
Yes, they're used by these tests: https://github.com/Automattic/wp-calypso/blob/4b685bbddf086e8841dc5fc1a909519da079d0fa/client/state/sites/domains/test/selectors.js#L28 Those tests, apparently compare `DOMAIN_PRIMARY` and `REST_API_SITE_DOMAIN_FIRST`. Honestly, I don't see the point of such tests, but :man_shrugging: They were there before, I modified them according to the changes in the rest of the PR.